<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fxloader</class>
 <widget class="QWidget" name="fxloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FX Elements Loader</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1041</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	font: 11pt &quot;Microsoft YaHei UI&quot;;
	border: 1px solid black;
	border-radius: 11px;
}

QGroupBox::title{
	left: 20px;
}</string>
   </property>
   <property name="title">
    <string>FX Elements Loader</string>
   </property>
   <widget class="QComboBox" name="show_cbx">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
font: 10pt &quot;MS Shell Dlg 2&quot;;
}
QListView{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="show_lbl_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Reference Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Seq</string>
    </property>
   </widget>
   <widget class="QComboBox" name="seq_cbx">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
font: 10pt &quot;MS Shell Dlg 2&quot;;
}
QListView{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="show_lbl_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Reference Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Shot</string>
    </property>
   </widget>
   <widget class="QComboBox" name="shot_cbx">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
font: 10pt &quot;MS Shell Dlg 2&quot;;
}
QListView{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="show_lbl_5">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Reference Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Task</string>
    </property>
   </widget>
   <widget class="QComboBox" name="task_cbx">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
font: 10pt &quot;MS Shell Dlg 2&quot;;
}
QListView{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="show_lbl_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Reference Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>SubTask</string>
    </property>
   </widget>
   <widget class="QComboBox" name="subtask_cbx">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
font: 10pt &quot;MS Shell Dlg 2&quot;;
}
QListView{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_btn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 11pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play_mov">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Play Mov</string>
    </property>
   </widget>
   <widget class="QLabel" name="show_lbl_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Reference Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="select_all">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Palatino Linotype&quot;;</string>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </widget>
  <widget class="QLabel" name="latest_version">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Palatino Linotype&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>1041</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="element_layout">
    <property name="leftMargin">
     <number>10</number>
    </property>
   </layout>
  </widget>
  <widget class="QPushButton" name="import_element">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>750</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>1051</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="comments">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>1041</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9.2pt &quot;Cambria Math&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="user_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Palatino Linotype&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
